resources/entrypoints/main.sh

```diff
@@ -40,13 +57,38 @@ else
 
 	set -- "$@" --nodaemon
 	if [ "$1" == "ndb_mgmd" ]; then
-		echo "[Entrypoint] Starting ndb_mgmd"
-		set -- "$@" -f $RONDB_DATA_DIR/config.ini --configdir=$RONDB_DATA_DIR/log
+		echo "[entrypoints/main.sh] Starting ndb_mgmd"
+		set -- "$@" -f "$RONDB_DATA_DIR/config.ini" --configdir="$RONDB_DATA_DIR/log"
```

Comment:
What about the permissions of the logs/data that the mgmd creates?

Reply:
The `umask` value of `0002` gets inherited by all processes started from this script, also `ndb_mgmd`, see PID `31` in this example:
```
docker exec mgmd_1 bash -c 'ps -fuxa; grep -H Umask /proc/*/status'
USER         PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
root          58  0.0  0.0   4360  3300 ?        Ss   07:04   0:00 bash -c ps -fuxa; grep -H Umask /proc/*/status
root          64  0.0  0.0   7060  1564 ?        R    07:04   0:00  \_ ps -fuxa
root           1  0.0  0.0   4360  3304 ?        Ss   07:02   0:00 /bin/bash ./docker_entrypoints/rondb_standalone/entrypoint.sh ndb_mgmd --ndb-nodeid=65 --initial
root          30  0.0  0.0   7236  4392 ?        S    07:02   0:00 sudo -E -u mysql /srv/hops/docker_entrypoints/rondb_standalone/main.sh ndb_mgmd --ndb-nodeid=65 --initial
mysql         31  0.8  0.0 1035104 25040 ?       Sl   07:02   0:01  \_ ndb_mgmd --ndb-nodeid=65 --initial --nodaemon -f /srv/hops/mysql-cluster/config.ini --configdir=/srv/hops/mysql-cluster/log
/proc/1/status:Umask:   0022
/proc/30/status:Umask:  0022
/proc/31/status:Umask:  0002
/proc/58/status:Umask:  0022
/proc/self/status:Umask:        0022
/proc/thread-self/status:Umask: 0022
```
Never had a problem with mgmd in testing. It appears that `ndb_mgmd` either does not create any files in the volumes or uses system default permissions, which will result in `0666 & ~UMASK = 0664` for files and `0777 & ~UMASK = 0775` for directories.